topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.md

```diff
@@ -0,0 +1,564 @@
+   C	DRR187559.2	1280	70|198	0:2 1279:5 0:29 |:| 0:52 1279:5 0:13 1279:3 0:23 1279:2 0:45 1280:1 0:9 1280:3 A:8
+   C	DRR187559.3	1279	106|73	0:4 1279:3 0:36 1279:4 0:10 1279:5 0:3 1279:5 0:2 |:| 0:39
+   C	DRR187559.4	1279	121|189	1279:6 0:17 1279:4 0:28 1279:2 0:30 |:| 0:7 1279:5 0:19 1279:5 0:20 1279:1 0:8 1279:1 0:1 1279:6 0:25 1279:2 0:44 A:11
+   C	DRR187559.5	1279	68|150	1279:2 0:20 1279:3 0:9 |:| 0:10 1279:3 0:24 1279:2 0:9 1279:5 0:21 1279:5 0:20 1279:5 0:9 1279:1 0:2
+   C	DRR187559.6	1280	137|246	0:2 1280:5 0:28 1279:1 0:28 1279:2 0:8 1279:2 0:23 1279:1 0:3 |:| 1279:1 0:2 1279:3 0:61 1279:2 0:14 1279:2 0:97 A:30 
+   ```
+
+   > <question-title></question-title>
+   >
+   > 1. Is the first sequence in the file classified or unclassified?
+   > 2. What is the taxonomy ID assigned to the first classified sequence?
+   > 3. What is the corresponding taxon?
+   >
+   > > <solution-title></solution-title>
+   > > 1. classified
+   > > 2. 1280
+   > > 3. 1280 corresponds to [Staphylococcus aureus](https://www.ncbi.nlm.nih.gov/datasets/taxonomy/tree/?taxon=1280) .
+   > {: .solution}
+   {: .question}
+
+- **Report**: tabular files with one line per taxon and 6 columns or fields
+
+   1. Percentage of fragments covered by the clade rooted at this taxon
+   2. Number of fragments covered by the clade rooted at this taxon
+   3. Number of fragments assigned directly to this taxon
+   4. A rank code, indicating
+      - (U)nclassified
+      - (R)oot
+      - (D)omain
+      - (K)ingdom
+      - (P)hylum
+      - (C)lass
+      - (O)rder
+      - (F)amily
+      - (G)enus, or
+      - (S)pecies
+
+      Taxa that are not at any of these 10 ranks have a rank code that is formed by using the rank code of the closest ancestor rank with a number indicating the distance from that rank. E.g., `G2` is a rank code indicating a taxon is between genus and species and the grandparent taxon is at the genus rank.
+
+   5. NCBI taxonomic ID number
+   6. Indented scientific name
+
+   ```
+   Column 1	Column 2	Column 3	Column 4	Column 5	Column 6
+   0.24 	1065 	1065 	U 	0 	unclassified
+   99.76 	450716 	14873 	R 	1 	root
+   96.44 	435695 	2 	R1 	131567 	cellular organisms
+   96.44 	435675 	3889 	D 	2 	Bacteria
+   95.56 	431709 	78 	D1 	1783272 	Terrabacteria group
+   95.53 	431578 	163 	P 	1239 	Firmicutes
+   95.49 	431390 	4625 	C 	91061 	Bacilli
+   94.38 	426383 	1436 	O 	1385 	Bacillales
+   94.04 	424874 	2689 	F 	90964 	Staphylococcaceae
+   93.44 	422124 	234829 	G 	1279 	Staphylococcus 
+   ```
+
+   > <question-title></question-title>
+   >
+   > 1. How many taxa have been found?
+   > 2. What are the percentage on unclassified?
+   > 3. What are the domains found?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > 1. 627, as the number of lines
+   > > 2. 0.24%
+   > > 3. Only Bacteria
+   > {: .solution}
+   >
+   {: .question}
+
+# Species extraction
```

Comment:
Maybe a sub-level. 
The structure could be : 
1. Read quality control and improvement
    1. Quality control
    2. Quality improvement  
3. Taxonomic profiling (better name to find)
    1. Taxonomic profiling
    2. Species extraction (not sure about the name: it might feel like we extract the reads associated to species - maybe better Species identification)
    3. Contamination identification

What do you think?